tests/integration_tests/build/test_dependencies.py

```diff
@@ -42,17 +42,17 @@ def test_num_dependencies(dep_file):
     # cargo tree displays a tree of dependencies which means
     # some of them will repeat. Below is a mechanism for filtering
     # unique dependencies.
-    # cargo tree tries to display a (*) at the end of each dependency that
-    # was already encountered but it does not do very well (libc appears
-    # multiple times).
+    # cargo tree tries to display a (*) at the end of each non-leaf dependency
+    # that was already encountered (crates without dependencies, such as libc
+    # appear multiple times).
     for line in deps:
         if line and "(*)" not in line:
             current_deps.add(line)
 
     # Use the code below to update the expected dependencies.
     # from pprint import pprint
     # with open(dep_file, "w", encoding='utf-8') as prev_deps:
-    #     pprint(sorted(current_deps), stream=prev_deps)
+    #      pprint(sorted(current_deps), stream=prev_deps)
```

Comment:
Intentional?

Reply:
Not really (I think it was me on auto-pilot added "hash"-"space" to every line but the final line used to not have a space?)